docs/how-to/build-cardano-sl-and-daedalus-from-source-code.md

```diff
@@ -61,7 +61,7 @@ Two steps remain, then:
 2.  Actually building the Cardano SL node (or, most likely, simply obtaining it
     from the IOHK's binary caches) can be performed by building the attribute `cardano-sl-static`:
```

Comment:
@rahilzebpay we should update description as well here

Reply:
@akegalj  Done!

Reply:
Hm, after another look at the line:
> Actually building the Cardano SL node...

I believe the correct binary that should be triggered here is `cardano-sl-node-static` (after a look at `release.nix`).

@jmitchell what is a difference between building `cardano-sl` and `cardano-sl-node-static` ? I believe the documentation is trying to give an example how to build a cardano node and later gives more examples on how to build other binaries (the later part of the documentation is also incorrect. For example there is a mention of `cardano-sl-wallet-static` which doesn't exist any more in our `release.nix`)